src/main/java/com/rev_connect_api/controllers/PartnerChannelController.java

```diff
@@ -0,0 +1,67 @@
+package com.rev_connect_api.controllers;
+
+import com.rev_connect_api.models.PartnerChannel;
+import com.rev_connect_api.services.PartnerChannelService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
+
+import java.net.URI;
+import java.util.List;
+
+@RestController
+@CrossOrigin(origins = "*")
+@RequestMapping("/partner_channels")
```

Comment:
```suggestion
@RequestMapping("/partner-channels")
```
No underscores in the path names por favor. It is an SEO thing, not that we care but this isn't common to do.